pjsip/include/pjsua2/media.hpp

```diff
@@ -1880,6 +1880,16 @@ class VideoWindow
      */
     void setFullScreen(bool enabled) PJSUA2_THROW(Error);
 
+    /**
+     * Set video window full-screen. This operation is valid only when the
+     * underlying video device supports PJMEDIA_VID_DEV_CAP_OUTPUT_FULLSCREEN
+     * capability. Currently it is only supported on SDL backend.
+     *
+     * @param mode		Fullscreen mode, see
+     *				pjmedia_vid_dev_fullscreen.
+     */
+    void setFullScreen2(pjmedia_vid_dev_fullscreen mode) PJSUA2_THROW(Error);
```

Comment:
Why in pjsua2 we create a new API to avoid api-break but not in pjsua?

Reply:
AFAIK in C++ assigning an integer value to an enum type is illegal, e.g: `setFullScreen2(PJ_TRUE)` will cause compile error. But it is legal in C, so no need to introduce a new API in pjsua.

Reply:
in pjsua, there will be API break though, is this okay?

Reply:
Existing pjsua app (using C, not C++) calling `pjsua_vid_win_set_fullscreen(PJ_TRUE/FALSE)` should not see any behavior change, so no API break?